internal/server/agentAdminEndpoint.go

```diff
@@ -93,6 +93,17 @@ func (gi *graphIntercept) WriteHeader(statusCode int) {
 	gi.originalHandler.WriteHeader(statusCode)
 }
 
+// Agent Admin GraphQL godoc
+// @id agent-proxy-admin
+// @tags Agent
+// @Summary This is the GraphQL Endpoint for administration of the agent proxy tokens.
```

Comment:
```Administration Endpoint of the Agent proxy tokens```

Minha sugestão aqui é tirar o `this` e o `GraphQL` e deixar o Agent em maíusculo pois entendo que se refere a um Agent específico, certo?

Reply:
Sim @tubedude ! Acho que como tá no título, acho que na descrição nao precisa ter que o endpoint é GraphQL boa :D